src/components/WorkspaceList/WorkspaceEmptyState.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
+import ErrorEmptyState from '../../shared/components/empty-state/ErrorEmptyState';
+
+const WorkspaceListView: React.FC = () => (
```

Comment:
Component is not named correctly. Also, just use the EmptyErrorState directly in WorkspaceListView. No need to create a separate component.